README.md

```diff
@@ -126,6 +126,7 @@ complete beginners.
 ### 2. Video Courses - Here are 2 from our range :
```

Comment:
@SahityaRoy , as you added a link, can you please change the `Here are 2 from our range` to `Here are 3 from our range`. Thank you.

Reply:
@SahityaRoy add 3 in description instead of two.